servicetemplates/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fservicetemplates/QHAna_w3-wip1/README.md

```diff
@@ -0,0 +1,24 @@
+# QHAna Docker [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This Service Template models the QHAna Framework in a Docker-based enrironment.
+
+**NOTE**: Currently, there are CORS issues when starting the application.
```

Comment:
Do they still exist?

Reply:
All cors issues should finally be solved

Reply:
Then I remove the note